tensorflow/core/kernels/cwise_ops_common.h

```diff
@@ -618,6 +618,14 @@ struct UnaryFunctor<CPUDevice, Functor> {
   }
 };
 
+template <typename Functor>
+struct UnaryFunctorWithArg<CPUDevice, Functor> {
+  void operator()(const CPUDevice& d, typename Functor::tout_type out,
+                  typename Functor::tin_type in, float val) {
```

Comment:
Can this be generalized too?

Reply:
Still waiting on this... if it can't be generalized, it will be better to restrict the change to within the `.*leaky_relu.*` source files.  Anything living within the generic  `cwise_ops_common.h` should be generally applicable.

Reply:
Hi @cantonios : sorry for the delay. I pushed the change just now. I have added Targ as "val" is independent of in_type & out_type. Let me know what you think.